frontend/src/components/core/ReportView/ReportView.tsx

```diff
@@ -25,13 +25,13 @@ import { ReportRunState } from "lib/ReportRunState"
 import { WidgetStateManager } from "lib/WidgetStateManager"
 
 import { ThemeProvider } from "baseui"
+import { ReportElement } from "lib/DeltaParser"
 import { mainWidgetTheme, sidebarWidgetTheme } from "lib/widgetTheme"
+
 import "./ReportView.scss"
 import "./Widget.scss"
 
-type SimpleElement = ImmutableMap<string, any>
-type Element = SimpleElement | BlockElement
-interface BlockElement extends List<Element> {}
+interface BlockElement extends List<ReportElement> {}
```

Comment:
Can't we just import this from `DeltaParser`?

Reply:
done